src/App.jsx

```diff
@@ -1,5 +1,6 @@
 import React from 'react';
 import './App.scss';
+import { Person } from './components/Person/Person';
```

Comment:
Ensure that the 'Person' component can handle cases where some properties, like 'age' for 'olya', are missing. This might require default values or conditional rendering within the 'Person' component.